README.MD

```diff
@@ -1,224 +1,277 @@
 # Manticore Search Helm Chart
 
-Version: 6.3.6.2
+**Version:** 6.3.6.2
 
-The helm chart is to be used to run Manticore Search in Kubernetes. It enables high availability by providing out-of-the-box replication and higher read throughput by automatic load balancing.
+The Manticore Search Helm Chart is designed to deploy and manage Manticore Search in Kubernetes. It ensures high availability with out-of-the-box replication and improves read throughput through automatic load balancing.
 
-### What this helm chart does
-* Deploys a StatefulSet with one or multiple Manticore Search pods (aka workers) that constitute a replication cluster.
-* Rolls out a Deployment with one or multiple (you can configure it in `values.yaml`) Manticore Search balancer instance.
-* Deploys needed services.
-* Provides Prometheus exporter for the running instances of Manticore Search.
+---
 
-### What you can do with the helm chart
-* You can create a table on any worker pod, add it to the cluster and it will be replicated to all the other pods.
-* You can scale the workers StatefulSet up and a new worker will be set up automatically to become a new replica in the cluster.
-* You can scale the workers StatefulSet down and the cluster will forget about the node.
-* If a worker is back up after a downtime it will rejoin the cluster and will be synced up to the active workers.
-* When you make a search request to the balancer it will route it to one of the workers.
-* [Data compaction](https://manual.manticoresearch.com/Securing_and_compacting_an_index/Compacting_an_index#Compacting-an-index) is done automatically.
 
+## Features
 
-# Installation
+### What This Helm Chart Does
+- Deploys a StatefulSet with one or multiple Manticore Search pods (workers) to form a replication cluster.
+- Rolls out a Deployment with one or multiple Manticore Search load balancer instances (configurable in `values.yaml`).
+- Deploys essential Kubernetes services.
+- Provides a Prometheus exporter for monitoring running Manticore Search instances.
 
-## Installation from Helm repo
+### Capabilities
```

Comment:
Let's add the new feature to the list - that you can read/write without the balancer using k8s built-in lb capabilities